nixos/tests/systemd.nix

```
@@ -108,6 +108,14 @@
       # Will not succeed unless ConditionFirstBoot=yes
       machine.wait_for_unit("first-boot-complete.target")
 
+      machine.wait_until_succeeds(
```

```suggestion
      machine.succeeds(
```
This should probably be `succeeds`, or else the test will stall instead of fail if the version does not match.